packages/manager/modules/pci/src/projects/project/storages/volume-backup/list/create-volume/controller.js

```diff
@@ -1,34 +1,20 @@
+import pick from 'lodash/pick';
+import BlockStorage from '../../../blocks/block.class';
 import { VOLUME_BACKUP_TRACKING } from '../../volume-backup.constants';
 
-const VOLUME_SIZE = {
-  MIN: 10,
-  MAX: 4000,
-};
-
 export default class VolumeBackupListCreateVolumeController {
   /* @ngInject */
-  constructor($translate, ovhManagerRegionService, VolumeBackupService) {
+  constructor($q, $timeout, $translate, VolumeBackupService) {
+    this.$q = $q;
+    this.$timeout = $timeout;
```

Comment:
```suggestion
  constructor($translate, VolumeBackupService) {
```
i don't see that $q or $timeout are used